core/web/testdata/flux-monitor-spec.toml

```diff
@@ -0,0 +1,28 @@
+type              = "fluxmonitor"
+schemaVersion       = 1
+name                = "example flux monitor spec"
+contractAddress   = "0x3cCad4715152693fE3BC4460591e3D3Fbd071b42"
+precision = 2
+threshold = 0.5
+absoluteThreshold = 0.0 # optional
+
+idleTimerPeriod = "1s"
+idleTimerDisabled = false
```

Comment:
This seems redundant and not idiomatic

Reply:
Spoke offline - it was user requested to have an explicit disable. Apparently it used to be just "not present = disabled" but that was consciously changed to this